generators/firechip/src/test/scala/ScalaTestSuite.scala

```diff
@@ -42,10 +42,9 @@ abstract class FireSimTestSuite(
   }
 
   def runTest(backend: String, name: String, debug: Boolean, additionalArgs: Seq[String] = Nil) = {
-    behavior of s"${name} running on ${backend} in MIDAS-level simulation"
     compileMlSimulator(backend, debug)
     if (isCmdAvailable(backend)) {
-      it should s"pass" in {
+      it should s"pass in ML simualtion on ${backend}" in {
```

Comment:
```suggestion
      it should s"pass in ML simulation on ${backend}" in {
```

Reply:
If CI passes you can just add a `[ci skip]` on the commit message to skip running CI on this. CI will run when everything is merged.